docs/nima/metrics/prometheus-exemplar-support.adoc

```diff
@@ -0,0 +1,25 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021, 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Metrics Support for Exemplars
```

Comment:
Similarly, we will probably support exemplars in a much different way in Nima.

Reply:
I have stripped all of the SE docs down to their basic components for Nima. I thought starting with the SE guide outlines/content might be easier, but I agree that could lead to more problems.